File: Models/Dog.cs
using System.Runtime.CompilerServices;
using System.Xml.Linq;

namespace DogBreedGuessingGame.Models
{
    public class Dog(String Name, String Breed, Uri PictureURL)
    {
        public String? Name { get; set; }
        public required String Breed { get; set; }
        public required Uri PictureURL { get; set; }
    }
}

File: Views/src/domain/Dog.ts
export class Dog {

    public name : String;
    private breed : String;
    public pictureURL : String;

    constructor(name : String, breed : String, pictureURL: String) {
        this.name = name;
        this.breed = breed;
        this.pictureURL = pictureURL;
    }

    public getBreed() : String {
        return this.breed;
    }

    public getSrc() : String {
        return this.pictureURL;
    }
}